src/program.js

```diff
@@ -84,8 +90,9 @@ export class Program {
   async execute(
     absolutePackageDir: string,
     {
-      systemProcess = process, logStream = defaultLogStream,
-      getVersion = defaultVersionGetter, shouldExitProgram = true,
+      checkForUpdates = defaultUpdateChecker, systemProcess = process,
+      logStream = defaultLogStream, getVersion = defaultVersionGetter,
+      shouldExitProgram = true, localEnv = WEBEXT_BUILD_ENV,
```

Comment:
Hmm, `localEnv` doesn't really make sense. I'm not sure why we called the last one that. I would suggest naming this `globalEnv` because that's what it is!